addons/website_slides/models/slide_slide.py

```diff
@@ -152,11 +152,21 @@ class Slide(models.Model):
     slide_views = fields.Integer('# of Website Views', store=True, compute="_compute_slide_views")
     public_views = fields.Integer('# of Public Views')
     total_views = fields.Integer("Total # Views", default="0", compute='_compute_total', store=True)
+    # comments
+    comments_count = fields.Integer('Number of comments', default=0, compute="_compute_comments_count", store=True)
```

Comment:
No need of default with compute :)

Moreover as it is not a critical field, I would not store it (used only in form view -> computed one by one, or in slide page view, one by one also).